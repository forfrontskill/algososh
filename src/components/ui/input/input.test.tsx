import renderer from 'react-test-renderer';
import { Input } from './input';

describe('Test Input component', () => {

    it('Input rendering', () => {
        const tree = renderer
            .create(<Input />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});