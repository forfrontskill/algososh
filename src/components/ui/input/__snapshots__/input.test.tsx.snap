// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Input component Input rendering 1`] = `
<div
  className="content "
>
  <input
    className="input text text_type_input text_color_input"
    data-testid="input"
    placeholder="Введите текст"
    type="text"
  />
</div>
`;
